### ISSUE TITLE
Passwords can appear in the log files

### ISSUE THREAD

**Issue by pallix** [liqd/adhocracy3.mercator#1407]
When registering with an already existing users, the REST error is logged and the password is logged in the log file. This sensitive information should not be stored in a clear-text format.

Example of the log:
```
WARNI [adhocracy_core.rest.views][MainThread] Found 1 validation errors in request: <b'{"content_type":"adhocracy_core.resources.principal.IUser","data":{"adhocracy_core.sheets.principal.IUserBasic":{"name":"lol"},"adhocracy_core.sheets.principal.IUserExtended":{"email":"lol@example.COM"},"adhocracy_core.sheets.principal.IPasswordAuthentication":{"password":"passwordlol2"}}}'>
```

**Comment by nidico** [comment]
A similar issue has been fixed in #698.

**Comment by pallix** [comment]
Fix by #1435
